pkg/autoscaler/collector_test.go

```diff
@@ -163,14 +163,17 @@ func TestMetricCollectorScraper(t *testing.T) {
 	mtp.ch <- now
 	var gotRPS, gotConcurrency, panicRPS, panicConcurrency float64
 	// Poll to see that the async loop completed.
-	wait.PollImmediate(10*time.Millisecond, 100*time.Millisecond, func() (bool, error) {
+	wait.PollImmediate(50*time.Millisecond, 100*time.Millisecond, func() (bool, error) {
```

Comment:
Did you **need** to change this or is this just for added resiliency?

Reply:
Ah too much debug output. Let me revert it.